### ISSUE TITLE
Import to Confluence breaks on links with an anchor

### ISSUE THREAD

**Issue by Lapsus** [dauxio/daux.io#208]
Having a link in a markdown file with an anchor works on generated and served files, but breaks the import to confluence.

Example:
```
[Automatische Setup](10_Entwicklungsumgebung/91_Verzeichnisstruktur_und_Repositories.md#page_Automatisches_Setup)
```

The error on the produces is:
In DauxHelper.php line 366:                                                                                                                    
Could not locate file '10_Entwicklungsumgebung/91_Verzeichnisstruktur_und_Repositories.md#page_Automatisches_Setup' 

If the anchor is removed, the import to confluence works normal.

**Comment by onigoetz** [comment]
Hi, I made a fix in the master branch that could solve your problem, can you try it before I release it ?

**Comment by Lapsus** [comment]
Sure, but im too busy this week. I can test it next week tough.

**Comment by onigoetz** [comment]
Hi, this has been released in [Version 0.14.0](https://github.com/dauxio/daux.io/releases/tag/0.14.0)
